wire-ios-sync-engine/Source/UserSession/ZMUserSession/ZMUserSession+Authentication.swift

```diff
@@ -85,6 +85,14 @@ extension ZMUserSession {
         completion()
     }
 
+    func close(deleteCookie: Bool) async {
+        await withCheckedContinuation { continuation in
+            close(deleteCookie: deleteCookie) {
+                continuation.resume()
+            }
```

Comment:
suggestion: make sure the continuation is never called twice
```suggestion
        await withCheckedContinuation { continuation in
            var resumed = false
            close(deleteCookie: deleteCookie) {
                guard !resumed else { return }
                resumed = true
                continuation.resume()
            }
```